coco/state.py

```diff
@@ -49,9 +51,19 @@ def __init__(
         """
         self.default_state_files = default_state_files
         self.exclude_from_reset = exclude_from_reset
+        self._storage_path = storage_path
+        self._name_active_state = "active"
+
+        # List saved states on disk
+        p = Path(self._storage_path).glob("**/*")
```

Comment:
The `Path()` here might be redundant since `_storage_path` seems to be guaranteed to be PathLike.

However, it also does not seem to hurt if it is there. =)

Reply:
Hm, I tried that, but `self._storage_path` was a `str`.